pages/pricing.vue

```diff
@@ -3,23 +3,35 @@
     <div class="text-center mt-5">
       <h1 class="pricing__heading">Pricing that feels just right</h1>
       <p class="pricing__subheading">Start with our free plan</p>
+
+      <div class="d-flex justify-content-center align-items-center plan_toggle__switch">
+        <span>Monthly</span>
+        <label class="switch">
+          <input type="checkbox" class="plan_toggle" v-model="isYearly">
+          <div class="slider round"></div>
+        </label>
+        <span>Yearly</span>
+      </div>
+
       <div
         class="d-flex align-items-center justify-content-center flex-wrap mt-4"
       >
-        <PricingCard category="Free" :amount="0" :features="freePlanFeatures" />
+        <PricingCard category="Free" :amount="0" :features="freePlanFeatures" :isYearly="isYearly" />
         <PricingCard
           category="Personal"
           :amount="13"
           :features="personalPlanFeatures"
           :isHighlighted="true"
+          :isYearly="isYearly"
         />
         <PricingCard
           category="Business"
           :amount="49"
           :features="businessPlanFeatures"
+          :isYearly="isYearly"
```

Comment:
We should simplify this and render 6 cards directly. 3 for monthly and 3 for yearly. Takes out any business logic from inside the component and just keeps them as an array or 2 here. 

So that in future if we have to change it just changing this data and that's all.